README.md

```diff
@@ -50,3 +50,7 @@ The sample catalog data is defined in [catalog.json](https://github.com/dotnet/e
 ### Contributing
 
 For more information on contributing to this repo, please read [the contribution documentation](./CONTRIBUTING.md) and [the Code of Conduct](CODE-OF-CONDUCT.md).
+
+### eShop on Azure
+
+The eShop application is designed to run locally. For a version of this app deployable on Azure, please view [the eShop on Azure](https://github.com/Azure-Samples/eShopOnAzure) repo.
```

Comment:
this makes it sound to me like dotnet/eshop is a toy for local dev box deployment only? I think you mean this one is designed to be deployed without explicit dependencies on any cloud platform, something like that, ?

Reply:
We've had a few issues trickle into the repo asking how to deploy this application so the comment here is an effort to explain that. I can update the description a bit.

```suggestion
For a version of this app configured for deployment on Azure, please view [the eShop on Azure](https://github.com/Azure-Samples/eShopOnAzure) repo.
```